src/features/Instructor/data/slice.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { RequestStatus } from 'features/constants';
+
+const initialState = {
+  instructorInfo: {
```

Comment:
Not sure about this, is kinda redundant `instructor.instructorInfo` I would consider to change it to `instructor.info`